src/Contexts/Adaptors/EVMAdaptors/EVMHomeAdaptorProvider.tsx

```diff
@@ -485,6 +499,41 @@ export const EVMHomeAdaptorProvider = ({
     }
   };
 
+  const initFallbackMechanism = async (
+    srcChainId: number,
+    destinationChainId: number,
+    recipient: string,
+    depositNonce: number,
+    decimalAmount: number
+  ): Promise<void> => {
+    const dstChainConfig = getСhainConfig(destinationChainId);
+    const { intervalMs } = getСhainTransferFallback(
+      srcChainId,
+      destinationChainId
+    );
+
+    setInterval(async () => {
+      const cereApi = await createCereApi(dstChainConfig.rpcUrl);
+      const { status } = await getBridgeProposalVotes(
+        cereApi,
+        srcChainId,
+        destinationChainId,
+        recipient,
+        depositNonce,
+        decimalAmount
+      );
+      console.log({ "Proposal votes status": status });
+      switch (status) {
+        case VoteStatus.APPROVED:
+          setTransactionStatus("Transfer Completed");
+          break;
+        case VoteStatus.REJECTED:
```

Comment:
For us is more important is to check either proposal `APPROVED` or `ACTIVE`. If `APPROVED` then the fallback can be stopped, if `ACTIVE` fallback should continue. If neither `APPROVED` nor `ACTIVE` the fallback should be stopped with an Error status.

Reply:
Resolved